content/projects/open-alias/index.md

```diff
@@ -19,57 +19,75 @@ show_on_projects: true
 allow_click_through: true
 percent_complete: 100
 github_url: "https://github.com/NAVCoin/navcoin-core/pull/213"
-project_url: "http://openalias.nav.community"
+project_url: "https://openalias.nav.community"
 reddit_url: ""
 twitter_url: ""
 card_color: "#7d5ab5"
 font_color: "#fff"
 weight: 100
 ---
```

Comment:
<img width="425" alt="image" src="https://user-images.githubusercontent.com/12672050/44697677-e794ac80-aad0-11e8-8aaf-8236e750e42f.png">

Currently shows like this on the projects page, you need to edit lines 15 and 16 in index.md so they say this.
```
completed: true
beta_testing: false
```

Reply:
Unless we are still considering this as being in beta